.env.example

```diff
@@ -0,0 +1,40 @@
+# Authentication
+# Package - https://next-auth.js.org/
+NEXTAUTH_URL=
```

Comment:
I think this can be `http://localhost:3000` for running locally

Reply:
Done